nload/plan.sh

```diff
@@ -0,0 +1,41 @@
+pkg_name=nload
+pkg_origin=core
+pkg_version="0.7.4"
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=("Apache-2.0")
+pkg_source="https://github.com/rolandriegel/${pkg_name}/archive/v${pkg_version}.tar.gz"
+pkg_shasum="a73b3a75356776860fc4c40daebce04c5022f73d39704a12fb0aeb88a751216a"
+pkg_build_deps=(core/make core/gcc core/automake core/autoconf)
+pkg_deps=(core/glibc core/gcc-libs core/ncurses)
+pkg_bin_dirs=(bin)
+
+do_prepare() {
+  ./run_autotools
+}
+
+do_after() {
```

Comment:
This isn't needed.  It will create the symlink inside the studio that the package is built in, but the artifact produced  doesn't package anything outside of its installed path (/hab/pkgs/<origin>/<name>/<version>/<timestamp>). 

If this is a requirement to use the package, for now we'll have to encode that in the README.